exonum-java-binding-core/src/test/java/com/exonum/binding/storage/indices/CheckedMapProofMultiMatcher.java

```diff
@@ -42,21 +42,36 @@ private CheckedMapProofMultiMatcher(List<MapTestEntry> entries) {
   @Override
   protected boolean matchesSafely(CheckedMapProof checkedMapProof) {
     ProofStatus status = checkedMapProof.getStatus();
+    List<MapEntry> presentEntries = checkedMapProof.getEntries();
```

Comment:
That’s another evidence that this API is not OK to use without a checked exception :upside_down_face: 

Please fix that with a regression test.